src/ExpandablePicker/ExpandablePicker.js

```diff
@@ -466,6 +466,11 @@ module.exports = kind(
 		}
 
 		if (this.getAutoCollapseOnSelect() && this.$.drawer.hasRendered && this.getOpen()) {
+			// this is needed due to the select and close delay, during which other history entries
+			// could have been pushed - we drop the entry immediately to prevent dropping the wrong
+			// entry later
+			this.removeHistoryEntry();
```

Comment:
This property is being checked here: https://github.com/enyojs/moonstone/pull/2691/files#diff-a273ea1fb0277d3dc6b4ae55158e2c63R391